CHANGELOG.md

```diff
@@ -1,3 +1,4 @@
 # 1.0.0
 ## Dokumentation
 - IFS-3928: Die Antora Dokumentation für das Bedienkonzept wurde in dieses Git-Repository umgezogen
+- IFS-3928: Die Grafiken in den Kapiteln *Bedienelement*, *Design Pattern* und *Use Cases* wurden mit dem neuen Labeln versehen
```

Comment:
IFS-3928: Die Antora-Dokumentation für das Bedienkonzept wurde in dieses Git-Repository migriert
IFS-3928: Die Grafiken in den Kapiteln *Bedienelement*, *Design Pattern* und *Use Cases* wurden mit den neuen Labeln versehen

Reply:
Done